graylog2-server/src/main/java/org/graylog2/contentpacks/facades/LookupTableFacade.java

```diff
@@ -76,7 +76,8 @@ private EntityDescriptor cacheDescriptor(String cacheId) {
     @VisibleForTesting
     Entity exportNativeEntity(LookupTableDto lookupTableDto, EntityDescriptorIds entityDescriptorIds) {
         final String tableId = entityDescriptorIds.get(EntityDescriptor.create(lookupTableDto.id(), ModelTypes.LOOKUP_TABLE_V1))
-                .orElseThrow(() -> new ContentPackException("Couldn't find lookup table entity " + lookupTableDto.id()));
+                .orElseGet(() -> entityDescriptorIds.get(EntityDescriptor.create(lookupTableDto.name(), ModelTypes.LOOKUP_TABLE_V1))
```

Comment:
The code is alright, but I just want to let you know about the slightly more idiomatic `Optional#or` which fits in quite nicely here:

```suggestion
                .or(() -> entityDescriptorIds.get(EntityDescriptor.create(lookupTableDto.name(), ModelTypes.LOOKUP_TABLE_V1)))
```

Reply:
thx. for the suggestion